packages/tradershub/src/features/cfd/constants.tsx

```diff
@@ -50,23 +50,30 @@ type TPlatformUrls = {
     };
 };
 
-export const CFDPlatforms = {
-    CFDS: 'CFDs',
-    CTRADER: 'ctrader',
-    DXTRADE: 'dxtrade',
-    MT5: 'mt5',
-} as const;
-
-export const MarketType = {
-    ALL: 'all',
-    FINANCIAL: 'financial',
-    SYNTHETIC: 'synthetic',
-} as const;
-
-export const Category = {
-    DEMO: 'demo',
-    REAL: 'real',
-} as const;
+export enum Category {
```

Comment:
@shontzu-deriv why are you change the type from object to enum? I'm pretty sure it's a bad idea. You can read about enums in TS [here](https://dev.to/ivanzm123/dont-use-enums-in-typescript-they-are-very-dangerous-57bh). Yeah, it's just one opinion, but I don't see any benefits to use unum by this way. You can ignore this comment and don't change this part of code, but anyway would be great if you can explain me your thoughts about this. Thanks a lot

Reply:
looking into the articles, actually I didn't really thought much of it except the way we're using it is like enums hence changing it accordingly